src/V3Force.cpp

```diff
@@ -148,15 +148,14 @@ class ForceState final {
     //  AstVar::user1p        -> ForceComponentsVar* instance (via m_forceComponentsVar)
     //  AstVarScope::user1p   -> ForceComponentsVarScope* instance (via m_forceComponentsVarScope)
     //  AstVarRef::user2      -> Flag indicating not to replace reference
-    //  AstVarScope::user3      -> AstVarScope*, a `valVscp` force component for each VarScope of
-    //  forced RHS
-    //  AstVarScope::user4p   -> AstNodeExpr*, the RHS expression
+    //  AstVarScope::user3p   -> AstNodeExpr*, the RHS expression
     const VNUser1InUse m_user1InUse;
     const VNUser2InUse m_user2InUse;
     const VNUser3InUse m_user3InUse;
-    const VNUser4InUse m_user4InUse;
     AstUser1Allocator<AstVar, ForceComponentsVar> m_forceComponentsVar;
     AstUser1Allocator<AstVarScope, ForceComponentsVarScope> m_forceComponentsVarScope;
+    std::unordered_map<const AstVarScope*, std::vector<AstVarScope*>> m_valVscps;
```

Comment:
```suggestion
    std::unordered_map<const AstVarScope*, std::unordered_set<AstVarScope*>> m_valVscps;
```

Presumably.  I don't see why we shouldn't be removing duplicates.